File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let sum = 0
    let runsum = []
    for(let i = 0; i < nums.length; i++){
        sum += nums[i]
        runsum.push(sum)
    }
    return runsum
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        int sum = 0;
        int[] runsum = new int[nums.Length];
        for(int i = 0; i < nums.Length;i++){
            sum += nums[i];
            runsum[i] = sum;
        }
        return runsum;
    }
}